src/robot.js

```diff
@@ -444,7 +444,7 @@ class Robot {
     app.use(express.query())
 
     app.use(express.json())
-    app.use(express.urlencoded({ limit, parameterLimit: paramLimit }))
+    app.use(express.urlencoded({ limit, parameterLimit: paramLimit, extended: false }))
```

Comment:
It looks like the default value is `true` (https://expressjs.com/en/api.html#express.urlencoded). Why change the value to `false`?

Reply:
Not sure why I chose `false` but I'm going to guess that I misunderstood the express docs which say (about `extended`): "Using the default is deprecated." I think they mean "not specifying a value and relying on the default value is deprecated" but I interpreted it as "The default is true and using that value is deprecated." 

At least that's my story now.

I'll switch it to `true`.